package-lock.json

```diff
@@ -59,92 +60,105 @@
         "@types/glob": "^7.2.0",
         "@types/mocha": "^8.2.3",
         "@types/mongodb": "^4.0.7",
-        "@types/node": "^12.20.42",
+        "@types/node": "^20.12.2",
         "@types/react": "^18.2.6",
         "@types/react-dom": "^18.2.4",
         "@types/tar": "^6.1.5",
         "@types/uuid": "^9.0.6",
-        "@types/vscode": "^1.63.1",
-        "@typescript-eslint/eslint-plugin": "^4.33.0",
-        "@typescript-eslint/parser": "^4.33.0",
-        "@vscode/test-electron": "^2.3.8",
+        "@types/vscode": "^1.87.0",
```

Comment:
Shouldn't it be 1.63.1 it self?(see package.json)

Reply:
nice find